templates/setup.html

```diff
@@ -260,14 +375,36 @@ <h3><b>{{ sublevel.name }}'s</b> examples</h3>
 
 
         <div class="row" style="margin-top: 30px;">
-          <div class="col-6">
+          {% if request.user.setup %}
+            <div class="col-3">
               <input onclick="forceTagsReset()" class="btn btn-primary col" type="reset" value="Reset"/>
-          </div>
-          <div class="col-6">
-            <input class="btn btn-primary col" type="submit" value="Submit"/>
-          </div>
-        </div>
+            </div>
 
+            <div class="col-3">
+              <input class="btn btn-primary col" type="button" id="loadButton" value="Load saved"/>
+            </div>
+
+            <div class="col-3">
+              <input class="btn btn-primary col" type="button" id="saveButton" value="Save"/>
+            </div>
+
+            <div class="col-3">
+              <input class="btn btn-primary col" type="submit" value="Submit"/>
+            </div>
+          {% else %}
+            <div class="col-4">
+              <input onclick="forceTagsReset()" class="btn btn-primary col" type="reset" value="Reset"/>
+            </div>
+
+            <div class="col-4">
+              <input class="btn btn-primary col" type="button" id="saveButton" value="Save"/>
+            </div>
+
+            <div class="col-4">
+              <input class="btn btn-primary col" type="submit" value="Submit"/>
+            </div>
```

Comment:
Could you explain more please. Because I'm not sure what's the issue.

Reply:
You duplicated this code:
```
            <div class="col-4">
              <input class="btn btn-primary col" type="button" id="saveButton" value="Save"/>
            </div>

            <div class="col-4">
              <input class="btn btn-primary col" type="submit" value="Submit"/>
            </div>
```
This one is in both cases. The only different is css class. I think in some version of bootstrap we can solve this one using `col` class instead of `col-<number>`, but I'm not sure if we support it. So we need to check it.

Reply:
Now I made it even better and less DRY.